README.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Mainboard communication
+
+The mainboard receives structs in the form:
+
+typedef struct Command { 
+  int16_t speed1; // Speed value for 1st motor
+  int16_t speed2; // Speed value for 2nd motor
+  int16_t speed3; // Speed value for 1rd motor
+  uint16_t throwerSpeed; // PWM for thrower motor
+  uint16_t delimiter; // 0xAAAA
+} Command;
+
+The mainboard returns feedback structs in the form:
+
+Typedef struct Feedback {
+  int16_t speed1; // encoder values from 1st motor
+  int16_t speed2; // encoder values from 2nd motor
+  int16_t speed3; // encoder values from 3rd motor
+  uint16_t delimiter; // 0xAAAA
+
+} Feedback;
```

Comment:
Improve formatting for better readability.

https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/

Reply:
improved readability